src/lib/utils.js

```diff
@@ -0,0 +1,49 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import {
+  ERROR_MESSAGE_NOT_INTEGER,
+  ERROR_MESSAGE_NOT_NUMBER,
+  ERROR_MESSAGE_NOT_POSITIVE_POSITIVE,
+} from './constants.js';
+
+export const readLineAsync = async message =>
+  MissionUtils.Console.readLineAsync(message);
+
+export const print = message => {
+  MissionUtils.Console.print(message);
+};
+
+export const pickNumberInRange = (start, end) =>
+  MissionUtils.Random.pickNumberInRange(start, end);
+
+export const splitIntoArray = (str, separator) => str.split(separator);
+
+export const getIsNumeric = num => {
+  if (typeof num !== 'number') return false;
+  return !Number.isNaN(num) && !Number.isNaN(parseFloat(num));
+};
+
+export const validatePositiveInteger = num => {
+  const isNumeric = getIsNumeric(num);
+  if (!isNumeric) throw new Error(ERROR_MESSAGE_NOT_NUMBER);
+
+  const isPositiveNumeric = num > 0;
+  if (!isPositiveNumeric) throw new Error(ERROR_MESSAGE_NOT_POSITIVE_POSITIVE);
+
+  const isInteger = Number.isInteger(num);
+  if (!isInteger) throw new Error(ERROR_MESSAGE_NOT_INTEGER);
+};
+
+export const getMapFilledZero = array =>
+  array.reduce((map, item) => map.set(item, 0), new Map());
```

Comment:
```suggestion
  new Map(array.map((item) => [item, 0]));
```
요렇게 해도 똑같이 동작할 것 같습니다! 이렇게 하면 조금더 `Map` 객체를 반환한다는 걸 명시해줄 수 있을 것 같다는 개인적인 생각에 제안드려봅니다 ☺️

Reply:
좋은 방법인 거 같아요 !! 감사합니다.